scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java

```diff
@@ -234,7 +235,7 @@ private boolean isRowSelected(int rowNum) {
             if (value != null) {
                 progress = Float.parseFloat(value.toString());
             }
-            int bx = new Double(Math.ceil(pw * progress)).intValue() - 601;
+            int bx = Double.valueOf(Math.ceil(pw * progress)).intValue() - 601;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the boxing to "Double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2153)